docker/addons/cassandra-writer/filters.toml

```diff
@@ -0,0 +1,11 @@
+# To listen all messsage broker subjects use default value "channels.>".
+# For specific filtering pass a list of subjects starting by "channels." and
+# followed by a subtopic (e.g ["channels.<channel_id>.sub.topic.x", ...]).
+[subjects]
+filter = ["channels.>"]
+
+# If writer is used with ContentType application/json JSON key-value pairs can
+# be filtered. Keys are used to store correspondinng pair. (e.g ["temperature", "humidity"]).
```

Comment:
this means that only the enlisted keys will be saved, right?

Reply:
Right!